### PR TITLE
[Pages] docusaurus site guide

### PR DESCRIPTION
**Pull request by deadlypants1973**
closes PCX-898

### CODE REVIEW

File: content/pages/framework-guides/deploy-a-docusaurus-site.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+pcx_content_type: how-to
+title: Deploy a Docusaurus site
+---
+
+# Deploy a Docusaurus site
+
+[Docusaurus](https://docusaurus.io) is a static-site generator. It builds a single-page application with fast client-side navigation, leveraging the full power of React to make your site interactive. It provides out-of-the-box documentation features but can be used to create any kind of site (personal website, product, blog, marketing landing pages, etc).
```
Comment (pedrosousa) [inline content/pages/framework-guides/deploy-a-docusaurus-site.md]:
```suggestion
[Docusaurus](https://docusaurus.io) is a static site generator. It builds a single-page application with fast client-side navigation, leveraging the full power of React to make your site interactive. It provides out-of-the-box documentation features but can be used to create any kind of site such as a personal website, a product site, a blog, or marketing landing pages.
```
Avoiding "etc."

File: content/pages/framework-guides/deploy-a-docusaurus-site.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+{{<render file="_tutorials-before-you-start.md">}}
+
+## Create a GitHub repository
+
+Create a new GitHub repository by visiting [repo.new](https://repo.new). When creating your repository, do not select to add a README file, .gitignore template or a license as these selections will cause the push to GitHub to fail.
+
+In your terminal, `cd` into your new Docusaurus project directory and run:
+
+```sh
+# Setup the local repository
```
Comment (pedrosousa) [inline content/pages/framework-guides/deploy-a-docusaurus-site.md]:
```suggestion
# Set up the local repository
```

File: content/pages/framework-guides/deploy-a-docusaurus-site.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+2. In **Account Home**, select **Pages** > **Create a project**. 
+3. Select the new GitHub repository that you created and, in the **Set up builds and deployments** section, select *Docusaurus* as your **Framework preset**. Your selection will provide the following information.
+
+{{<table-wrap>}}
+
+| Configuration option | Value                       |
+| -------------------- | --------------------------- |
+| Production branch    | `main`                      |
+| Build command        | `npm run build`             |
+| Build directory      | `build`                    |
```
Comment (pedrosousa) [inline content/pages/framework-guides/deploy-a-docusaurus-site.md]:
```suggestion
| Build directory      | `build`                     |
```

File: content/pages/framework-guides/deploy-a-docusaurus-site.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+{{</table-wrap>}}
+
+4. Go to **Environment variables (advanced)** and create a variable name `NODE_VERSION` with a value of `16`.
```
Comment (pedrosousa) [inline content/pages/framework-guides/deploy-a-docusaurus-site.md]:
```suggestion
4. Go to **Environment variables (advanced)** and create a `NODE_VERSION` variable with a value of `16`.
```

File: content/pages/framework-guides/deploy-a-docusaurus-site.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+After deploying your site, you will receive a unique subdomain for your project on `*.pages.dev`.
+
+Every time you commit new code to your Docusaurus site, Cloudflare Pages will automatically rebuild your project and deploy it. You will also get access to [preview deployments](/pages/platform/preview-deployments/) on new pull requests and be able to preview how changes look to your site before deploying them to production.
```
Comment (pedrosousa) [inline content/pages/framework-guides/deploy-a-docusaurus-site.md]:
```suggestion
Every time you commit new code to your Docusaurus site and push those changes to GitHub, Cloudflare Pages will automatically rebuild your project and deploy it. You will also get access to [preview deployments](/pages/platform/preview-deployments/) on new pull requests and be able to preview how changes look to your site before deploying them to production.
```

**Review by pedrosousa — APPROVED** [review]

File: content/pages/framework-guides/deploy-a-docusaurus-site.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+| Configuration option | Value                       |
+| -------------------- | --------------------------- |
+| Production branch    | `main`                      |
+| Build command        | `npm run build`             |
+| Build directory      | `build`                     |
+
+{{</table-wrap>}}
+
+4. Go to **Environment variables (advanced)** and create a `NODE_VERSION` variable with a value of `16`.
```
Comment (deadlypants1973) [inline content/pages/framework-guides/deploy-a-docusaurus-site.md]:
```suggestion
4. Go to **Environment variables (advanced)** > **Add variable** > create a `NODE_VERSION` variable with a value of `16`.
```

**Pull request merged**